core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/BCFile.java

```diff
@@ -634,6 +635,7 @@ public long getRawSize() {
       // backwards compatibility
       if (version.equals(API_VERSION_1)) {
         LOG.trace("Found a version 1 file to read.");
+        decryptionParams = new NoFileEncrypter().getDecryptionParameters();
```

Comment:
Should probably just call "this.decrypter.getDecryptionParameters()" after the next line instead of constructing "NoFileEncrypter" twice.

Reply:
The NoFileEncrypter class is only used to get the "no crypto" decryption params.  The next line uses NoFile**D**ecrypter and stores that locally to pass around.

Reply:
Oh my mistake. I misread both objects as the same class.